libs/design/src/molecules/paginator/paginator.component.html

```diff
@@ -0,0 +1,21 @@
+<nav role="navigation" attr.aria-label="Pagination Navigation">
+  <a attr.aria-label="Go to Previous Page" tabindex="0" (click)="onNotifyPrevPageChange()" *ngIf="showPrevPageChange()" class="daff-paginator__prev">
+    <fa-icon [icon]="['fa', 'chevron-left']"></fa-icon>Prev
```

Comment:
Do we need a `peerDependency` on `@fortawesome/font-awesome-angular`?

Reply:
Added the peerDependency